modules/custom/localgov_restricted_content/localgov_restricted_content.info.yml

```diff
@@ -0,0 +1,5 @@
+name: "Essex Restricted Content"
```

Comment:
Do I remember correctly that the module is likely to end up as an LGD module? If so, should the `description` and `package` name reflect that now?